src/scripts/main.js

```diff
@@ -2,7 +2,40 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
 
-// write your code here
+people.forEach(person => {
+  const newRow = document.createElement('tr');
+
+  const nameCell = document.createElement('td');
+
+  nameCell.textContent = person.name;
+  newRow.appendChild(nameCell);
+
+  const sexCell = document.createElement('td');
```

Comment:
The task requires the column header to be 'gender', not 'sex'. Change the variable name to 'genderCell' and the content to 'person.gender'.